core-api/src/main/java/com/optimizely/ab/config/ProjectConfig.java

```diff
@@ -73,6 +76,7 @@ public String toString() {
     private final String revision;
     private final String version;
     private final boolean anonymizeIP;
+    private final Boolean botFiltering;
```

Comment:
Please use boolean? not Boolean.

Reply:
I am using Boolean instead of boolean because we need to handle null case that if botFilltering is not in datafile than it should be null so only Boolean is nullable.